Composer/packages/client/src/components/Header.tsx

```diff
@@ -191,18 +188,16 @@ export const Header = () => {
   };
   const toggleGetStarted = (newvalue) => {
     hideTeachingBubble();
+    setShowGetStartedTeachingBubble(false);
     setShowGetStarted(newvalue);
   };
 
   // pop out get started if #getstarted is in the URL
   useEffect(() => {
-    if (location.hash === '#getstarted') {
-      setShowGetStartedTeachingBubble(true);
+    if (showGetStartedTeachingBubble) {
       setShowGetStarted(true);
```

Comment:
Are there two states to track `showGetStartedTeachingBubble`? Would it be simpler to just use the recoil state as the source of truth instead of maintaining two states?

Reply:
There are not two states for `showGetStartedTeachingBubble`. The recoil state value  is the source of truth for the teaching bubble within the getStarted pane.

The confusion is from the other local state value (showTeachingBubble) this is a local state value that controls a different teaching bubble (the teaching bubble that points the user to the start bot feature in the header).

To avoid confusion I have renamed this local state variable so we have `showStartBotTeachingBubble` and `showGetStartedTeachingBubble` respectively.